environment/service_test.go

```diff
@@ -39,10 +41,16 @@ objects:
 func TestGetAllTemplatesForAllTypes(t *testing.T) {
```

Comment:
Let's add a check in SetupSuite (so, it's run before any test) that version/commit vars are set (!= "") before you set them to test values in your tests.

Reply:
Never mind. We won't have these vars set in tests. So, can't test it here.